keyboards/e88/info.json

```
@@ -0,0 +1,109 @@
+{
+    "keyboard_name": "e88",
+    "url": "",
+    "maintainer": "2-n",
+    "width": 17,
+    "height": 6,
+    "layouts": {
+        "LAYOUT": {
```

File contents are fine here, but this string needs to be an exact match for the name of the layout macro to which it applies:

https://github.com/qmk/qmk_firmware/blob/a88884870160f620c1e9161d32982701de301864/keyboards/e88/e88.h#L30

```suggestion
        "LAYOUT_all": {
```